### DISCUSSION TITLE
Unable to install app

### DISCUSSION THREAD

**Discussion by darkrain88** [beclab/Olares#745] [Q&A]
Unable to install app

![image](https://github.com/user-attachments/assets/5f73f179-83ff-4412-9251-c0d7fe2ca7df)
in grey  how to

**Comment by TShentu** [comment]
Olares seems unable to get your GPU info. Which OS version are you using?
Can you post the response from api https://market.your-olaresid.olares.com/app-store/v1/cluster/resource
like the following
![image](https://github.com/user-attachments/assets/61ace251-f1af-490a-92ce-c268a26824ff)

**Reply by darkrain88** [reply]
![image](https://github.com/user-attachments/assets/c2f013d7-b9f8-489d-a595-2196b9a0e68e)

**Reply by hysyeah** [reply]
Hello!  Could you execute this command `kubectl logs app-service-0 -n os-system > app-service.log`  and paste app-service.log here? This my help us solve this problem. thx!

**Reply by darkrain88** [reply]
[app-service.log](https://github.com/user-attachments/files/18125896/app-service.log)

**Reply by hysyeah** [reply]
thx! We've resolved the issue. you can manually update app-service image to `beclab/app-service:0.2.60`.

**Reply by darkrain88** [reply]
how to

**Reply by fnalways** [reply]
Hi @darkrain88 , if you are not sure about what to do:
1. Open Control Hub (or 控制面板 if your system language is Chinese), and select Browse > System > os-sytem > Statefulsets > app-service > Edit Yaml.
![image](https://github.com/user-attachments/assets/354babae-1e68-4f25-bde7-41b5d49c36bd)
2. In this YAML file, locate `beclab/app-service: {version number}`, and update it to `0.2.60`.
![image](https://github.com/user-attachments/assets/aec65495-e627-4c14-b54a-29c5874b3b4c)
3. Click OK to apply the change and restart the service.
When you see a green indicator, it means the image is successfully updated.
![image](https://github.com/user-attachments/assets/1264a8c0-ed0e-433e-9ac4-02e8be4ed815)

Let me know if you need further help!

**Reply by darkrain88** [reply]
many thanks.

![image](https://github.com/user-attachments/assets/6584f677-45af-4051-a3ce-6b9e1c525579)
some of them still require gpu support
![image](https://github.com/user-attachments/assets/1a871f8c-9c89-4ad8-9667-cce68af7e6b4)


"Will passing through my Intel iGPU from Proxmox VE to an Ubuntu VM help solve this issue?"

**Reply by fnalways** [reply]
> Will passing through my Intel iGPU from Proxmox VE to an Ubuntu VM help solve this issue?

Currently no. However, we plan to release an updated version of Ollama next week, which will allow machines without a GPU to install and use it. In the meantime, are there any other applications you'd like to use but are unable to? We'd be happy to look into them as well.

**Reply by darkrain88** [reply]
![image](https://github.com/user-attachments/assets/d2b3ecfb-f6b0-468a-8845-d621e158b493)
 in grey

**Reply by PDCem** [reply]
> > Will passing through my Intel iGPU from Proxmox VE to an Ubuntu VM help solve this issue?
> 
> Currently no. However, we plan to release an updated version of Ollama next week, which will allow machines without a GPU to install and use it. In the meantime, are there any other applications you'd like to use but are unable to? We'd be happy to look into them as well.

I am not using Proxmox but i cant install the ollama on Olares, coz he say my GPU ist bad or not enough, so did you release a fix ? 
i want it install and use the Cloud modells and i know it works coz i try the Olares on my gaming pc with a nvidia 4070 and there i can install and also i can tell him to use only cloud modells. 
Next week is many moths ago, so did you fix that, if yes, why im getting allways the same error, i hate that. 
Please fix that.

**Comment by fnalways** [comment]
Hi @darkrain88 - Could you please let us know which device you installed Olares on (Mac, windows, etc.)?

**Reply by darkrain88** [reply]
root@server:~# uname -a
Linux server 6.8.0-50-generic #51-Ubuntu SMP PREEMPT_DYNAMIC Sat Nov  9 17:58:29 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
root@server:~# ls
ls           lsb_release  lshw         lsirq        lsmem        lsof         lspower
lsattr       lscpu        lsinitramfs  lslocks      lsmod        lspci        lsusb
lsblk        lsfd         lsipc        lslogins     lsns         lspgpot
root@server:~# lscpu
Architecture:             x86_64
  CPU op-mode(s):         32-bit, 64-bit
  Address sizes:          39 bits physical, 48 bits virtual
  Byte Order:             Little Endian
CPU(s):                   4
  On-line CPU(s) list:    0-3
Vendor ID:                GenuineIntel
  BIOS Vendor ID:         QEMU
  Model name:             Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
    BIOS Model name:      pc-i440fx-9.0  CPU @ 2.0GHz
    BIOS CPU family:      1
    CPU family:           6
    Model:                142
    Thread(s) per core:   1
    Core(s) per socket:   4
    Socket(s):            1
    Stepping:             10
    BogoMIPS:             4607.99

Debian 12 under PVE

**Comment by darkrain88** [comment]
![image](https://github.com/user-attachments/assets/b4c8a4c7-8e39-4bbb-83e7-6043458b60b4)
when i get some app, it shows problem of storage, please help to indicate how to .

**Reply by fnalways** [reply]
From the error message, it seems there isn't enough disk space for the app. I’ve passed this issue to the developers to identify possible causes and solutions.

**Reply by hysyeah** [reply]
@darkrain88 Could you run the command `kubectl logs app-service-0 -n os-system > app-service.log` again and provide us with an app-service.log? This will help us determine whether it's truly a disk space issue or something else.

**Reply by darkrain88** [reply]
[app-service.log](https://github.com/user-attachments/files/18135812/app-service.log)
attached.

**Reply by hysyeah** [reply]
Sorry for the delayed response.
 We installed Olares on PVC Debian12 but couldn't reproduce your issue. It might be that Prometheus is unable to retrieve system metrics properly or something else.Could you please provide more information?
Use the following command to get all pod log.
`tar -czvf log.tgz /var/log/pods`

**Reply by darkrain88** [reply]
![image](https://github.com/user-attachments/assets/a0134ab9-478c-4823-acff-4a57a8da7bd1)
maybe conflict with  my Promethues. as above 

log  file   link  https://s.fnnas.net/s/09db8733853d4b77a3

，pwd：123456，

**Reply by hysyeah** [reply]
The root cause of the issue is that the `node-exporter` you installed conflicts with the port(hostPort:9100) used by Olares' `node-exporter`, causing the Olares `node-exporter` cannot start up, which in turn prevents Prometheus from retrieving system resource information.

Solution:

You can either uninstall your Prometheus or modify the port of your node-exporter

**Reply by darkrain88** [reply]
ill try
